adapter/slack.py

```diff
@@ -112,3 +113,22 @@ async def list_bots(self, request: Request):
             return Response(status_code=200)
         except SlackApiError as e:
             raise HTTPException(status_code=400, detail=f"Slack API Error : {e}")
+    
+    def event_message(self, event, say):
+        if 'thread_ts' in event:

```

Comment:
What does this line if code handling ?

Reply:
Here's the link, specifically look at the bug.
https://api.slack.com/events/message/message_replied

It's for checking whether the received message is a reply for a thread or not.

Reply:
There's also a similar explanation here https://api.slack.com/messaging/retrieving#finding_threads